docs/docs/dev_docs/getting_started/sandbox.md

```diff
@@ -379,3 +379,8 @@ Our complete output should now be:
 ```
 
 That's it! We have successfully deployed a private token contract to an instance of the Aztec network and mined private state-transitioning transactions. We have also queried the resulting state all via the interfaces provided by the contract.
+
+## Next Steps
+Here we showed how to interact with the sandbox, but didn't go into details on how to write your own contract or any relevant setup needed for it. 
+
+You can find details on setting up [here](./noir_contracts.md) and refer to the [Contracts section](../contracts/) on syntax, compiling, deploying and interacting with how to start writing contracts.
```

Comment:
The link is wrong. Remember to build it and check locally. We should probably add the build step to CI.
```suggestion
You can find details on setting up [here](./noir_contracts.md) and refer to the [Contracts section](../contracts/main.md) on syntax, compiling, deploying and interacting with how to start writing contracts.
```